frame/clearing-house/src/lib.rs

```diff
@@ -2150,8 +2148,7 @@ pub mod pallet {
 				};
 				let last_oracle_10bp =
 					last_oracle.try_div(&T::Decimal::saturating_from_integer(1000))?;
-				oracle = math::clip(
-					oracle,
+				oracle = oracle.clamp(
```

Comment:
Haha, nice, that looks better too!

Reply:
rereading this now - I would change `oracle` here to `oracle_price`